code/modules/mob/living/carbon/xenomorph/Castes/Hivelord.dm

```diff
@@ -43,9 +43,10 @@
 	// *** Ranged Attack *** //
 	spit_delay = 1.3 SECONDS
 	spit_types = list(/datum/ammo/xeno/sticky)
+	armor_deflection = 5
```

Comment:
undocumented change

Reply:
#904 I accidentally overrode changes made here. So I had to re-enter them. The values are the same.